lib/ballot/ballot.go

```diff
@@ -68,6 +68,20 @@ func (b Ballot) String() string {
 }
 
 func (b Ballot) IsWellFormed(networkID []byte, conf common.Config) (err error) {
+	if err = b.isBallotWellFormed(networkID, conf); err != nil {
+		return
+	}
+
+	if b.Vote() != VotingEXP {
```

Comment:
I really don't think we should skip signature verification like this. We should be able to carry over a payload with the right signature.

Reply:
> I really don't think we should skip signature verification like this. We should be able to carry over a payload with the right signature.

@Geod24 Then, do you have any good idea for signing expired ballot? :)

Reply:
Let's discuss IRL